src/Instrument/Transformer/CachingTransformer.php

```diff
@@ -87,10 +96,11 @@ public function transform(StreamMetaData $metadata)
         }
 
         $originalUri = $metadata->uri;
-        $cacheUri    = str_replace($this->rootPath, $this->cachePath, $originalUri);
+        $cacheUri    = $this->kernel->getCachePathResolver()->getCachePathForResource($originalUri);
 
         $lastModified  = filemtime($originalUri);
-        $cacheModified = file_exists($cacheUri) ? filemtime($cacheUri) : 0;
+        $cacheExisted = file_exists($cacheUri);
```

Comment:
Ok, thank you, will fix the alignment.
As for the name - it reflects that cache file existed before file_put_contents call, should I rename it anyway?